test-reports/TEST-tests.hfmagpsu.HfmagpsuTests-20170821153916.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
```

Comment:
Test reports shouldn't be in the repo. Please remove these files and add a .gitignore to ignore them in future (copy one from a similar submodule)